grafana-plugin/src/containers/RotationForm/ShiftSwapForm.tsx

```diff
@@ -0,0 +1,211 @@
+                  <WithConfirm title="Are you sure to delete shift swap request?" confirmText="Delete">
+                    <IconButton variant="secondary" tooltip="Delete" name="trash-alt" onClick={handleDelete} />
+                  </WithConfirm>
+                </WithPermissionControlTooltip>
+              )}
+              <IconButton variant="secondary" className={cx('drag-handler')} name="draggabledots" />
+              <IconButton name="times" variant="secondary" tooltip="Close" onClick={onHide} />
+            </HorizontalGroup>
+          </HorizontalGroup>
+
+          <div className={cx('fields')}>
+            {id !== 'new' && (
+              <Field label="Creator">
+                <Input disabled={readOnly} value={beneficiaryName}></Input>
+              </Field>
+            )}
+
+            <HorizontalGroup height="auto">
+              <Field label="Swap start">
+                <DateTimePicker
+                  timezone={store.currentTimezone}
+                  disabled={readOnly}
+                  value={dayjs(shiftSwap.swap_start)}
+                  onChange={handleShiftSwapStartChange}
+                />
+              </Field>
+              <Field label="Swap end">
+                <DateTimePicker
+                  timezone={store.currentTimezone}
+                  disabled={readOnly}
+                  value={dayjs(shiftSwap.swap_end)}
+                  onChange={handleShiftSwapEndChange}
+                />
+              </Field>
+            </HorizontalGroup>
+
+            <Field label="Description">
+              <TextArea rows={4} disabled={readOnly} value={shiftSwap.description} onChange={handleDescriptionChange}>
+                {shiftSwap.description}
+              </TextArea>
+            </Field>
+            {id !== 'new' && (
+              <Field label="Taken by">
+                {shiftSwap?.benefactor ? (
+                  <UserItem
+                    pk={shiftSwap?.benefactor}
+                    shiftColor={SHIFT_SWAP_COLOR}
+                    shiftStart={shiftSwap.swap_start}
+                    shiftEnd={shiftSwap.swap_end}
+                  />
+                ) : (
+                  <Text type="secondary">Not taken yet</Text>
+                )}
+              </Field>
+            )}
+          </div>
+
+          <HorizontalGroup justify="space-between">
+            <Text type="secondary">Current timezone: {getTzOffsetString(dayjs().tz(currentTimezone))}</Text>
+            <HorizontalGroup>
+              {id === 'new' ? (
+                <WithPermissionControlTooltip userAction={UserActions.SchedulesWrite}>
+                  <Button variant="primary" onClick={handleCreate}>
+                    Create
+                  </Button>
+                </WithPermissionControlTooltip>
+              ) : (
+                <WithPermissionControlTooltip userAction={UserActions.SchedulesWrite}>
```

Comment:
Since it's the same `WithPermissionControlTooltip`, might be worth adding it on top of the current jsx (below `<HorizontalGroup>`)